cmd/csaf_checker/reporters.go

```diff
@@ -157,7 +157,7 @@ func (r *tlsReporter) report(p *processor, domain *Domain) {
 func (r *tlpWhiteReporter) report(p *processor, domain *Domain) {
 	req := r.requirement(domain)
 	if !p.badWhitePermissions.used() {
-		req.message(InfoType, "No advisories labeled TLP:WHITE tested for accessibility.")
+		req.message(InfoType, "No access-protected advisories labeled TLP:WHITE found.")
```

Comment:
I guess we should separate the 2 cases in future: see #402...

Reply:
As this is already a new issue I would say this should not hinder the merge of this PR.

Reply:
That was the intention :-) Therefore, I created a separate issue.